src/main/java/com/iota/iri/Milestone.java

```diff
@@ -187,33 +205,32 @@ private boolean validateMilestone(SpongeFactory.Mode mode, TransactionViewModel
                                 transactionViewModel2.trits(), 0, index, NUMBER_OF_KEYS_IN_A_MILESTONE);
                         if (testnet || (new Hash(merkleRoot)).equals(coordinator)) {
                             new MilestoneViewModel(index, transactionViewModel.getHash()).store(tangle);
-                            return true;
+                            return VALID;
+                        } else {
+                            //invalid Milestone
```

Comment:
Why need this comment as your return shows it is `INVALID`.

Reply:
thx, will remove.